recipes/home-assistant/webview.js

```diff
@@ -1,12 +1,12 @@
 module.exports = Ferdium => {
```

Comment:
Is this file change required in this PR? Or was it a defect that's being fixed from a previous PR? (like eslint)?

Reply:
cool .tx